data-raw/variable_reference.csv

```diff
@@ -446,3 +446,22 @@ lon,radonb,LONGITUDE
 shortref,radonb,REFERENCE
 comment,radonb,
 ,radonb,ID
+labnr,nerd,LabID,
+c14age,nerd,CRA,
+c14std,nerd,Error,
+c13val,nerd,DC13,
+material,nerd,Material,
+site,nerd,SiteName,
+country,nerd,Country,
+feature,nerd,SiteContext,
+lat,nerd,Latitude,
+lon,nerd,Longitude,
+comment,nerd,Comments,
+shortref,nerd,Source,
+,nerd,DateID,
+,nerd,OthLabID,
+,nerd,Problems,
+,nerd,Species,
+,nerd,SiteID,
+,nerd,SiteType,
```

Comment:
Our c14bazAAR datatype has a predefined column for sample species and type of site (https://github.com/ropensci/c14bazAAR/blob/master/data-raw/variable_definition.csv). So maybe we could map these variables as well? Or do they have a very different meaning in NERD?

Reply:
Missed those two, thanks for noticing.